src/lib.rs

```diff
@@ -487,7 +412,7 @@ impl IntoRawFd for VsockStream {
 
 impl Drop for VsockStream {
     fn drop(&mut self) {
-        unsafe { close(self.socket) };
+        _ = close(self.socket);
```

Comment:
`let _` is better, same reason with above

Reply:
Done.